buildtools/win/redirect-cc/redirect-cc.cc

```diff
@@ -0,0 +1,125 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
```

Comment:
do we have to keep the binary in the source tree?

Reply:
I debated that myself as well, but it is pretty small (113K) and we'd have to put something in place to build it before building anything else. If we do want to build it locally, maybe the setup can be left for a follow up?

Reply:
Addressed in 8698b649419d14a5344d3170fb6d5d1f1f69acc5